### PR TITLE
Fix latest download count, re: issue #6741

### PR DESCRIPTION
**Pull request by SteveWilkes**
Updated fix for issue #6741 from PR #6742.

Displays the latest listed package download count, or no package download count if there are no listed packages.

### CODE REVIEW

**Comment by chenriksson** [comment]
@SteveWilkes Thanks for your contribution!

I think a more consistent experience would be to always show this version-specific download count, but default to the current version instead of the latest. By default, `DisplayPackage.cshtml` displays the latest listed version but could show an unlisted version if no listed are available.

This would simplify the logic to just:
```
@Model.DownloadCount.ToNuGetNumberString() @(Model.DownloadCount == 1 ? "download" : "downloads")	
                    of current version
```

**Comment by scottbommarito** [comment]
I think we need to ask the question "what do users want to see here" and then answer that before we go forward with this.

I think we may also want to consider showing the download count of
- the latest stable version
- the most popular version (the one with the most downloads)

I tend to agree with @chenriksson that "current version" is probably the most useful thing to show there, however.

**Comment by SteveWilkes** [comment]
Thanks for the feedback! I've updated the PR to use `Model.DownloadCount` as requested.

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 1]
```diff
@@ -750,7 +750,7 @@
                 </li>
                 <li>
                     <i class="ms-Icon ms-Icon--Giftbox" aria-hidden="true"></i>
-                    @Model.PackageVersions.First().DownloadCount.ToNuGetNumberString() @(Model.PackageVersions.First().DownloadCount == 1 ? "download" : "downloads")
+                    @Model.DownloadCount.ToNuGetNumberString() @(Model.DownloadCount == 1 ? "download" : "downloads")
                     of latest version
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: This needs to be updated to say "current version".

Comment (SteveWilkes) [reply src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: Done! :)

**Comment by chenriksson** [comment]
Verified and merged... thanks @SteveWilkes!

**Pull request merged**